bsg_test/bsg_dramsim3.cpp

```diff
@@ -156,6 +156,17 @@ class BSGDRAMSim3 {
         _memory_system->ClockTick();
     }
 
+    /////////////////
+    // Print Stats //
+    /////////////////
+    void printTagStats(uint32_t tag) {
+        _memory_system->PrintTagStats(tag);
+    }
+
+    void printFinalStats() {
+        _memory_system->PrintStats();
+    }
```

Comment:
What's the purpose of this? I see it called below

Reply:
this will print the stats for the final partial epoch, it also does a little bit of final formatting to close out the json file
https://github.com/bsg-external/DRAMsim3/blob/2b8042d4d2161f4d1d1dcb373577e3fbd23b533d/src/dram_system.cc#L72

Reply:
Ahh. So it's an enhancement for DramSim3

Reply:
It was always there in DramSim3, just was never getting called, so i added it and modified to include the tag stats